src/qt/walletview.cpp

```diff
@@ -197,6 +200,77 @@ void WalletView::gotoVerifyMessageTab(QString addr)
         signVerifyMessageDialog->setAddress_VM(addr);
 }
 
+void WalletView::gotoLoadPSBT()
+{
+    QString filename = GUIUtil::getOpenFileName(this,
+        tr("Load Transaction Data"), QString(),
+        tr("Partially Signed Transaction (*.psbt)"), nullptr);
+    if (filename.isEmpty()) return;
+    std::ifstream in(filename.toLocal8Bit().data(), std::ios::binary);
+    // https://stackoverflow.com/questions/116038/what-is-the-best-way-to-read-an-entire-file-into-a-stdstring-in-c
+    std::string data(std::istreambuf_iterator<char>{in}, {});
```

Comment:
Would adding a bound check (max filesize) make sense at this point?

Reply:
Agree. Lame example `ln -s /dev/zero /tmp/u.psbt` and then open /tmp/u.psbt.

Reply:
Added. You'd think check the file size trivial in C++...